include/mbedtls/aes.h

```diff
@@ -69,26 +69,50 @@ extern "C" {
 // Regular implementation
 //
 
+/** VIA padlock needs 16 extra bytes for alignment. */
+#if (defined(MBEDTLS_PADLOCK_C) && defined(MBEDTLS_HAVE_X86)) ||        \
+    (defined(MBEDTLS_AESNI_C) && MBEDTLS_AESNI_HAVE_CODE == 2)
+#define MBEDTLS_PADLOCK_EXTRA 16
+#else
+#define MBEDTLS_PADLOCK_EXTRA 0
+#endif
+
+/** Each round key occupies 16 bytes. */
```

Comment:
This comment is just wrong, and we all managed to miss this while discussing `MBEDTLS_AES_RK_UINT32S`!

Reply:
The round keys are 16 `uint32_t`s, which is 128 bytes each. They aren't 16 bytes. We were discussing the units that `LEN` would be in - some assuming bytes - and a completely wrong comment nearby, which would have confused someone less familiar with AES, was completely ignored.

This comment should be corrected!